AudioLink/Shaders/AudioLink.cginc

```diff
@@ -199,7 +199,7 @@ float4 AudioLinkGetAmplitudeAtFrequency(float hertz)
 float AudioLinkGetAmplitudeAtNote(float octave, float note)
 {
     float quarter = note * 2.0;
-    return AudioLinkLerpMultiline(ALPASS_DFT + float2(octave * AUDIOLINK_EXPBINS + quarter, 0));
+    return AudioLinkLerpMultiline(ALPASS_DFT + float2(octave * AUDIOLINK_EXPBINS + quarter, 0)).r;
```

Comment:
Good catch, but this is bad. The function should just return float4 like the above one

Reply:
will address this in the next commit